lxd/storage_volumes.go

```diff
@@ -84,6 +84,10 @@ func storagePoolVolumesGet(d *Daemon, r *http.Request) Response {
 				return InternalError(err)
 			}
 			volume.UsedBy = volumeUsedBy
+
+			if strings.Contains(volume.Name, "/") {
+				volume.Type = fmt.Sprintf("%s (snapshot)", volume.Type)
+			}
```

Comment:
That's an actual API break so I don't think it's reasonable to do this here. The exact same check in the CLI tool would be totally fine however.